lib/srv/db/elasticsearch/util.go

```diff
@@ -37,50 +37,52 @@ func GetQueryFromRequestBody(e common.EngineConfig, contentType string, body []b
 		Knn   any `json:"knn" yaml:"knn"`
 	}
 
+	log := e.Log.With("content-type", contentType)
```

Comment:
nit: inline attributes are preferred unless `log` will be reused.

Reply:
It is reused, though?